citi-onboarding/Aqua-Jr
# Hotfix/footer

## What I did
- Fixed the alignment on the right side
- Removed props and placed the map inside the Footer component
- Adjusted the spacing between the phone number and the Phone title
- Removed the About Us console
## How to test
- Access the  ***hotfix/footer*** branch through your terminal
- Run **yarn start** on one terminal and **nodemon index.js** on another one
- Go to the localhost 3000 page on your browser and press F12
- Now check the following:
  - Access the **console** tab and make sure there aren't any **console.log** commands running
  - Now, make sure the website is on its web version and observe the alignment on the far right (the right border of the LinkedIn icon should be vertically aligned with the i on CITi)
  - Compare the spacing between Telefone and the phone number to the one on the other elements and make sure they're the same.
